### DISCUSSION TITLE
foundry installation error in Windows

### DISCUSSION THREAD

**Discussion by khandelwalmoksh** [Cyfrin/foundry-full-course-cu#2518] [Q&A] [answered]
Tried installing as Administrator.
![Screenshot (4)](https://github.com/user-attachments/assets/7ea41294-351a-4466-b131-fd4887fc83e0)

**Comment by EngrPips** [comment, marked as answer]
Hello @khandelwalmoksh, Please read this article https://medium.com/@regodefies/foundry-installation-on-windows-10-309407e39dee see if it helps you install Foundry.

**Comment by 48cork** [comment]
I got same error and I cant compile as well

**Reply by EngrPips** [reply]
I use Mac, so I know nothing about this Window issue, Let's hope someone who knows what's wrong sees this discussion and chimes in to help.

**Comment by mumernisar** [comment]
Hey @48cork .
You have to install WSL2 for windows..!! Then use that command in Ubuntu. Then it will work. You can find a video guide on how to do that on Cyfrin's Foundry Fundamentals course. It's the one of the first videos.
If you can't find lemme know I'll link it for ya

**Comment by EngrPips** [comment]
@48cork  Please try @mumernisar  suggestion and let us know if that get you unblocked.

**Comment by 48cork** [comment]
Yes, it worked.

Em sáb., 21 de set. de 2024 11:17, EngrPips ***@***.***>
escreveu:

**Reply by mumernisar** [reply]
Of course it did 😛♥️

**Reply by EngrPips** [reply]
Interesting, Good job guys

**Comment by 48cork** [comment]
Thanks for the tip!

Em sáb., 21 de set. de 2024 12:01, EngrPips ***@***.***>
escreveu:

**Reply by EngrPips** [reply]
You should mark this discussion as answered by @mumernisar

**Comment by 48cork** [comment]
how?

**Reply by mumernisar** [reply]
You have to be the owner of this discussion to do that I think.
So no worries.
@khandelwalmoksh will see to it if he sees this and his issue is resolved.

**Reply by EngrPips** [reply]
> You have to be the owner of this discussion to do that I think. So no worries. @khandelwalmoksh will see to it if he sees this and his issue is resolved.

Indeed

**Comment by 48cork** [comment]
Ok, thanks.
